### PR TITLE
Updates to README and bugfix in 'read-configuration' action

### PR DESCRIPTION
**Pull request by JamesDawson**
Fleshed-out the top-level README and updated/added examples.

The `read-configuration` action was supposed to emit the raw JSON as well as the individual values, but wasn't doing so - Pester test added for this scenario.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,78 @@
 ![build](https://github.com/endjin/dependabot-pr-parser/workflows/build/badge.svg)
 [![GitHub license](https://img.shields.io/badge/License-Apache%202-blue.svg)](https://raw.githubusercontent.com/corvus-dotnet/Corvus.Deployment/master/LICENSE)
 
+This repo contains 3 docker-based GitHub actions that can be used to enable continuous integration for a configurable subset of dependencies being managed by GitHub Dependabot.
```
Comment (HowardvanRooijen) [inline README.md:6]: What type of dependencies does it support? NPM? Python? .NET?

File: README.md [hunk 2]
```diff
@@ -3,10 +3,78 @@
...
+
+## Overview
```
Comment (HowardvanRooijen) [inline README.md:8]: Is it worth adding a section that points to projects that are making use of this, and thus you can show people what common json configuration files look like?

File: README.md [hunk 3]
```diff
@@ -3,10 +3,78 @@
...
+
+Whilst tools like [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) simplify the process of testing updates to a repository's package dependencies, there can still be signficant coordination work to apply such changes and have the resulting updates cascade through the rest of your software estate.
+
+The GitHub actions detailed below provide the plumbing to enable the following features:
+1. The ability to identify which dependencies are being updated and the 'scale' of the update, in terms of the [semantic version](https://semver.org) increment
+1. A policy-based approach for managing which dependencies are approved for being automatically merged
+1. The ability to batch multiple, auto-mergeable dependency updates into a single release 'event'
+
```
Comment (HowardvanRooijen) [inline README.md:16]: Could you include the workflow diagram in the repo, so it can be displayed in the readme?

**Comment by MikeLarah** [comment]
We should include a link to the pr-autoflow workflow documentation somewhere: https://github.com/endjin/.github/blob/master/workflow-templates/README.md

File: README.md [hunk 4]
```diff
@@ -3,10 +3,78 @@
...
+In addition, the action emits dynamic outputs based on the keys from the input JSON document.
+
+### Example Usage
+
+Given the following configuration file exists in the repo.
+```json
+{
+  "setting1": "some value",
+  "settign2": "another value"
```
Comment (MikeLarah) [inline README.md:43]: Typo in `settign2`

**Review by HowardvanRooijen — APPROVED** [review]

**Pull request merged**